client/ayon_deadline/abstract_submit_deadline.py

```diff
@@ -11,6 +11,7 @@
 from datetime import datetime
 from copy import deepcopy
 from typing import Optional
+import clique
 
```

Comment:
Shouldn't `clique` be part of the non-builtins?
```suggestion

import clique
```